tests/serializers/tests.py

```diff
@@ -348,6 +348,18 @@ def test_serialize_proxy_model(self):
         self.assertEqual(base_data, proxy_data.replace('proxy', ''))
         self.assertEqual(base_data, proxy_proxy_data.replace('proxy', ''))
 
+    def test_deserialized_object_class_repr(self):
+        author = Author.objects.create(name='John Doe')
```

Comment:
Ohh ya, I don't need 3 models. Previously I thought the `m2m_data` param is required.
I haven't heard in memory model, is it okay if I only use author model??
I used string interpolation because I don't know the id from newly created model.